<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RobocupULaval Competition</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QComboBox" name="defis_comboBox">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabWidgetPage1">
         <attribute name="title">
          <string>Description</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Description</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="description_label">
               <property name="frameShape">
                <enum>QFrame::Box</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Description du défi Description du défi Description du défi Description du défi Description du défi  Description du défi Description du défi Description du défi Description du défi Description du défi  Description du défi Description du défi Description du défi Description du défi Description du défi  Description du défi Description du défi Description du défi Description du défi Description du défi  Description du défi Description du défi Description du défi Description du défi Description du défi  Description du défi Description du défi Description du défi Description du défi Description du défi  Description du défi Description du défi Description du défi Description du défi Description du défi  Description du défi Description du défi Description du défi Description du défi Description du défi  Description du défi Description du défi Description du défi Description du défi Description du défi  Description du défi Description du défi Description du défi Description du défi Description du défi  Description du défi Description du défi Description du défi Description du défi Description du défi  Description du défi Description du défi Description du défi Description du défi Description du défi  Description du défi Description du défi Description du défi Description du défi Description du défi  &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="textFormat">
                <enum>Qt::AutoText</enum>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="margin">
                <number>10</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabWidgetPage2">
         <attribute name="title">
          <string>Score</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Défi:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
